atlas-fastq-provider-post-install-tests.sh

```diff
@@ -34,6 +36,17 @@ setup() {
     fi
 }
 
+@test "Download Fastq file via AWS S3" {
+    if  [ "$resume" = 'true' ] && [ -f "$fastq_file_ftp" ]; then
```

Comment:
```suggestion
    if  [ "$resume" = 'true' ] && [ -f "$fastq_file_s3" ]; then
```
should it be like that?